### ISSUE TITLE
Set up local develop environment for manual harvester testing 

### ISSUE THREAD

**Issue by kimwdavidson** [GSA/datagov-ckan-multi#195]
# Description

It is useful to be able to test harvesting jobs manually, on a local machine. In order to support this, the new harvesters need to ship a docker compose configuration that not only sets up a running harvester, but also a running CKAN configured and ready for use as a target for the data that is harvested.

# Acceptance criteria
- [x] Document a Docker Compose command that will set up the harvester dependencies, *and*, CKAN and its dependencies
  - This may require additional configuration and code to support the smooth operation of this local environment
- [ ] Verify that @adborden can follow the instructions and run the datajson harvester

For reference, ( cc @adborden ), [this is a base docker compose setup](https://github.com/okfn/docker-ckan) that many of our team (and other orgs) use for local CKAN development.

**Comment by kimwdavidson** [comment]
Paul has full context and will follow up to spec this ticket

**Comment by avdata99** [comment]
@akariv @adborden 

Developed at [a new branch](https://gitlab.com/datopian/ckan-ng-harvest/tree/66_add_CKAN_container) 
[MR created](https://gitlab.com/datopian/ckan-ng-harvest/merge_requests/72)

Some instructions added to [README.md](https://gitlab.com/datopian/ckan-ng-harvest/blob/66_add_CKAN_container/README.md) file

**Comment by akariv** [comment]
@avdata99 seems like a lot of duplication with ckan-cloud-docker.
What was wrong with the work that you've already done there?

**Comment by kimwdavidson** [comment]
@adborden to review, then circle with @akariv and @avdata99

**Comment by kimwdavidson** [comment]
Adam to decide whether to merge PR, once merged, move to QA and assign to Aaron

**Comment by kimwdavidson** [comment]
@avdata99 making changes based on adam's review

**Comment by avdata99** [comment]
Branch 66 still not merged. 
We still have some pendings definition at the [PR](https://github.com/datopian/ckan-ng-harvest/pull/6)

**Comment by akariv** [comment]
There are still a lot of CR issues and comments which were not resolved
yet...
Waiting for @avdata99 to complete the CR fixes before we can merge this PR.

**Comment by avdata99** [comment]
Fixed many of these reviews. The last one is to define the CKAN imagen to use (OKF or Viderum CKAN cloud docker)

**Comment by kimwdavidson** [comment]
@avdata99 finished work from reviews on Friday. @akariv to review changes and merge, move to QA and assign to @adborden

**Comment by avdata99** [comment]
Moving to QA since the PR is reviewed and merged

**Comment by avdata99** [comment]
Assigned to @adborden to QA. 
We can run this together Aaron if you prefer.

**Comment by adborden** [comment]
@avdata99 when I start the docker containers, CKAN complains that it cannot connect to the database due to an invalid password.

```
ckan_1          | psql: FATAL:  password authentication failed for user "ckan"
```

**Comment by avdata99** [comment]
I update the images (I had custom ones)

```
docker pull viderum/ckan-cloud-docker:db-latest
docker pull viderum/ckan-cloud-docker:ckan-latest-datagov-theme
```

Now I'm able to reproduce the error
